tests/npm-tsoa-app/README.md

```diff
@@ -0,0 +1,19 @@
+# Node.js/npm CCF app using tsoa
```

Comment:
I am wary of writing more docs in Markdown files that reference other Markdown files. Could we move all the build-your-JS-app docs to a new dedicated section [here](https://microsoft.github.io/CCF/master/developers/index.html) (not necessarily in this PR!)?

Reply:
Tracked as #1622